spyder/plugins/ipythonconsole/widgets/help.py

```diff
@@ -125,5 +125,4 @@ def _handle_inspect_reply(self, rep):
             if content.get('status') == 'ok' and content.get('found', False):
                 signature = self.get_signature(content)
                 if signature:
-                    self._control.show_calltip(_("Arguments"), signature,
-                                               signature=True, color='#2D62FF')
+                    self._control.show_calltip(signature, color='#2D62FF')
```

Comment:
This is what is being displayed on the console, but it seems to come from another source (not the LSP)

Reply:
Yes, it comes from the kernel. The important thing is that both calltips (Editor and Console ones) have the same look and feel.

Reply:
Ok, (for now) the `console` one will not display highlighted parameter, nor the docstrings.

Reply:
Ok, the console works differently and changing things there to adjust its calltips to the look and feel of Editor is out of the scope of this PR.